pkg/selinux/setroubleshoot-app.js

```diff
@@ -0,0 +1,7 @@
+require([
+            "selinux/setroubleshoot",
+        ],
+function(troubleshoot) {
+    const app = document.getElementById('app');
+    troubleshoot.init(app);
+});
```

Comment:
> Seems like this file should be merged with setroubleshoot.js ... that's what I've been doing with all the other places where we're moving code out of the HTML due to CSP.

On the other hand, this way we have cleaner separation. I noticed I haven't removed the jquery dependency in setroubleshoot.js, but I could. That way it could be included from anywhere, with any top level html element.
I guess the question is whether to keep this glue between the actual component and the rendering target or merge them like you did before. I like the separation, but for the sake of having fewer files I could go with the merged variant.